docs/get-started/README.md

```diff
@@ -38,11 +38,17 @@ breadcrumb: ''
 
 ## Wallets
 
+Filecoin wallets allow you to manage Fil, Filecoin's native token. Wallets store the private keys that allow you to authorize Filecoin transactions, including paying for storage deals and sending Fil to other accounts. See the [Wallet Addresses reference](/reference/#wallet-addresses) to learn about the kinds of addresses used by Filecoin accounts.
```

Comment:
this typically is all caps. would be great if we decided that one way or another, and had a style guide that gave that type of guidance for all our docs.

Reply:
er, by this i mean Fil vs FIL. this is what i get for quickly dropping comments between meetings, sorry!

Reply:
> and had a style guide that gave that type of guidance for all our docs.

It turns out [we do](https://docs.filecoin.io/community/contribute/grammar-formatting-and-style/#grammar-and-spelling) and I just didn't know about it 😄

Reply:
wow, 🆒.